### ISSUE TITLE
Documentation improvement

### ISSUE THREAD

**Issue by marrek-az** [medcv/OneEyeFaceDetection#1]
This looks like a fascinating project, which I'm attempting to try out. The Readme is a bit thin in places, and I'd like to help correct that once I understand what I'm missing. In particular, the section on deploying to DeepLens points to an AWS page about deploying the sample projects, and I'm having trouble reproducing the steps you used to deploy your code. I have the lambda function created, but can you describe in more detail what you did to deploy the Deeplens project?

**Comment by FSurani** [comment]
Agree. Was wondering if there was more information here on how to do this? @medcv

**Comment by AJibiki** [comment]
このReadmeに記載されていることだけでも（週末の３日掛かりましたが）、十分に動作させることができました。
要は、シンプルに考えることです。

エンジニアとして、やり方は自分で考えトライアンドエラーで対応すべきだと思いますので、僕も詳細情報は公開はしませんね。
http://hiscomputingdevblog.hatenadiary.jp/entry/2020/03/20/110358
